templates/example_fiber_segmentation.ipynb

```diff
@@ -0,0 +1,190 @@
+{
```

Comment:
<p>We should run this with the example dataset so the use can see what the output looks like</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>